modules/compliance-operator-install.adoc

```diff
@@ -25,4 +25,11 @@ For more information about restarting Sensor, see "Restarting Sensor in the secu
 
 . Select the *Compliance Operator* to view the details page.
 
-. Read the information about the Operator, and then click *Install*.
\ No newline at end of file
+. Read the information about the Operator, and then click *Install*.
```

Comment:
I'm not sure if this is too technical but maybe it would be good to add a note warning explaining that if ACS is installed before the Compliance Operator, then sensor will restart the connection with central. This will be reflected in the logs but should be seamless enough; thus I'm not sure if makes sense to add it.

Reply:
afaik, Sensor restart is not applicable for 4.5 and later versions, and we have removed it from the documentation. @dashrews78 Could you please pitch in on this one?

Reply:
Sensor's pod does not restart but the connection between sensor and central does. This is reflected in sensor's and central's logs.

Reply:
@lvalerom @agantony  I think that is too technical a detail.  I would leave that out.  At least that is my opinion

Reply:
Makes sense to me 👍